api/node/cover.md

```diff
@@ -152,7 +152,7 @@ The types used for properties in .slint design markup each translate to specific
 | `int` | `Number` | |
 | `float` | `Number` | |
 | `string` | `String` | |
-| `color` | {@link Color} |  |
+| `color` | {@link RgbaColor} |  |
```

Comment:
This seems correct to me, but the other type links are wrong, aren't they? `image` is now `SlintImageData`, not `ImageData`?

Reply:
No. I used the interface types here. Because that is what you set and also what you get. 

* interface `Brush`: class `SlintBrush`
* interface `ImageData`: class `SlintImageData`
* interface `RgbaColor`: class `SlintRgbaColor`

Reply:
You're right, the use of the interface is correct in this table. Apologies :)

Reply:
Nothing to apologies for ;-)